### ISSUE TITLE
Oad reports added required parameter when it just moved from path to operation

### ISSUE THREAD

**Issue by mikekistler** [Azure/openapi-diff#220]
In [this PR](https://github.com/Azure/azure-rest-api-specs/pull/17296), oad reports [many instances of "AddingRequiredParameter"](https://github.com/Azure/azure-rest-api-specs/pull/17296#issuecomment-1008963541) that are false positives, because the parameter was present and required in the previous version, but was specified at the path level rather than in the operation, as it is in the new version.

While this is a change in the way the API is described, it is not actually a change in the API definition, so should not be reported as a breaking change.

**Comment by jianyexi** [comment]
This is not false alarm,  removing the path level parameters is also breaking change

**Comment by mikekistler** [comment]
In this case, path parameters were neither added nor removed. They were simply described in a different place.

**Comment by mikekistler** [comment]
Not sure what is going on -- I've tried to get a simple recreate for this problem but so far I am not successful.

**Comment by mikekistler** [comment]
I think I may see what is going on.  The PR was updated but the PR checks were not marked as "outdated" or updated in place.  This makes it look like the _current code_ has problems when in fact the problems were fixed.

If this is correct then we can close this issue but should open one on the pipeline to update the PR checks section in place.
